packages/front-end/components/Forms/CodeTextArea.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { ReactNode } from "react";
+import Field, { FieldProps } from "./Field";
+import AceEditor from "react-ace";
+import "ace-builds/src-noconflict/mode-sql";
+import "ace-builds/src-noconflict/mode-javascript";
+import "ace-builds/src-noconflict/mode-python";
+import "ace-builds/src-noconflict/mode-yaml";
+import "ace-builds/src-noconflict/mode-json";
+import "ace-builds/src-noconflict/theme-textmate";
+
+export type Props = Omit<
+  FieldProps,
+  "value" | "onChange" | "options" | "multi" | "initialOption"
+> & {
+  syntax: string;
+  placeholder?: string;
```

Comment:
I don't think you need this prop here. Since you're are inheriting FieldProps it should be included already.

Reply:
I removed the explicit `placeholder` prop from the types. When testing, it appears that I do have to pass the `placeholder` prop into the `AceEditor`. Passing it in to the `Field` component doesn't render the placeholder. Let me know if this is what you are referring to or if you like the explicit option better.